.gitmodules

```diff
@@ -3,7 +3,8 @@
 	url = https://github.com/lf-lang/reactor-c.git
 [submodule "org.lflang/src/lib/cpp/reactor-cpp"]
 	path = core/src/main/resources/lib/cpp/reactor-cpp
-	url = https://github.com/lf-lang/reactor-cpp
+    url = https://github.com/lf-lang/reactor-cpp.git
+    branch = "connection-optimization"
```

Comment:
Why change the module configuration? Usually we just change the ref of the submodule